### PR TITLE
feat: adding peers filters to docs

### PR DESCRIPTION
**Pull request by leofj**

### CODE REVIEW

**Comment by anton-delphai** [comment]
CI is red

File: delphai_api/types.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ async def parse_subfields(request: Request, root_field: field_model = Depends())
     "headquarters",
     Optional[str],
     ["continent", "country", "state", "city"],
-    description="Headquarters filter. Available options: continent, country, state, city.",
+    description="Headquarters filter. Available options: continent, country, state, city.",  # noqa
```
Comment (anton-delphai) [inline delphai_api/types.py:49]:
```suggestion
    description="Headquarters filter. Available options: continent, country, state, city.",  # noqa: E501
```

**Comment by anton-delphai** [comment]
Oh wow, the deployment is failing
https://docs-ps-1809-updating-peers.delphai.pink/

**Comment by leofj** [comment]
> Oh wow, the deployment is failing https://docs-ps-1809-updating-peers.delphai.pink/

It's working now

**Review by anton-delphai — APPROVED** [review]

**Pull request merged**